repository/src/interpret/create.rs

```diff
@@ -124,6 +124,7 @@ pub async fn create_agenda(
         timestamp: get_timestamp(),
         transactions_hash: Agenda::calculate_transactions_hash(&transactions),
         height: last_header.height + 1,
+        previous_block_hash: last_header.previous_hash,
```

Comment:
I think `last_header.to_hash256()` is correct. This is pointing to the second to last block.

Reply:
reflected